autogen.pl

```diff
@@ -1481,6 +1481,11 @@ sub in_tarball {
     $m4 .= "m4_define([package_libevent], [1])\n";
     $m4 .= "m4_define([libevent_tarball], [" . $libevent_tarball . "])\n";
     $m4 .= "m4_define([libevent_directory], [" . $libevent_directory . "])\n";
+
+    # Grab the unique configure options from each of the 3rd party packages
+    # JJH Re-enable
+    #system("./config/extract-3rd-party-configure.pl -p \"3rd-party/".$libevent_directory."/\" -n \"Libevent\" -m > config/auto-libevent.m4");
```

Comment:
I know it's commented out, but I'm not sure we should support this for libevent.  We auto-add a bunch of arguments to our internal libevent (in part, to make sure we don't pull in even more dependencies, like suddenly having an SSL dependency).  I'd be really leery of supporting pulling trying to support all those configurations of libevent.  Similar argument with hwloc, although we don't pre-set nearly as many options there.

Add to that, we intentionally don't expand the tarball in autogen.pl, so that makes it doubly hard.

Reply:
I agree. I'll remove in the next push.